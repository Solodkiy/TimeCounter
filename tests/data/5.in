14,00 - 15,40
16.08 - 16.28