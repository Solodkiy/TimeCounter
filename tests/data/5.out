14:00 - 15:40 = 1:40
16:08 - 16:28 = 0:20
                ----
                2:00